doc/content/lorawan/concentrators/_index.md

```diff
@@ -0,0 +1,95 @@
+
+This page provides a quick description of each of the LoRa concentrators.
+
+## SX1301
+
+{{< figure src="SX1301.jpg" class="float plain" >}}
+
+The SX1301 is Semtech's first outdoor LoRaWAN concentrator, designed to provide worldwide LoRaWAN gateway capability in ISM bands. It is the first SoC to integrate Semtech's LoRa Concentrator IP.
+
+- Up to -142.5dBm sensitivity with SX1257 Tx/Rx front-end
+- 70 dB CW interferer rejection at 1 MHz offset
+- Able to operate with negative SNR, CCR up to 9dB
+- Emulates 49x LoRa demodulators and 1x (G)FSK demodulator
+- Dual digital TX&RX radio front-end interfaces
+- 10 programmable parallel demodulation paths
+- Dynamic data-rate (DDR) adaptation 
+- True antenna diversity or simultaneous dual-band operation
+
+The SX1301 is succeeded by the SX1302.
+
+Read more about the SX1301 on Semtech's SX1301 [product page](https://www.semtech.com/products/wireless-rf/lora-core/sx1301).
+
+## SX1302
+
+{{< figure src="SX1302.jpg" class="float plain" >}}
+
+The SX1302 succeeds the SX1301. It reduces current consumption, has better thermal capability, is cheaper, and can handle more traffic than its predecessor.
+
+- Up to -141 dBm sensitivity with SX1250 Tx/Rx front-end
+- 125 kHz LoRa reception with:
+    - 8 x 8 channels LoRa® packet detectors
+    - 8 x SF5-SF12 LoRa® demodulators
+    - 8 x SF5-SF10 LoRa® demodulators
+- 125 /250 / 500 kHz LoRa® demodulator
+- (G)FSK demodulator
+- Direct interface to Semtech transceivers
+- SX1255, SX1257 and SX1250
+- Single 32 MHz clock
+
+The SX1302 is succeeded by the SX1303.
+
+Read more about the SX1302 on Semtech's SX1302 [product page](https://www.semtech.com/products/wireless-rf/lora-core/sx1302).
+
+## SX1303
+
+{{< figure src="SX1303.jpg" class="float plain" >}}
+
+The SX1303 succeeds the SX1302, and is Semtech's most modern LoRa concentrator. It is pin and size compatible to its predecessor, and further reduces current, improves thermal capability, and is cheaper. In addition, it supports Time Difference On Arrival (TDOA) geolocation via a new Fine Timestamp capability.
+
+- Fine Timestamp
+- Up to -141dBm sensitivity with SX1250 Tx/Rx front-end
+- 125kHz LoRa reception with:
+    - 8 x 8 channels LoRa packet detectors
+    - 8 x SF5-SF12 LoRa demodulators
+    - 8 x SF5-SF10 LoRa demodulators
+    - Up to 8 packets can be received simultaneously when Fine Timestamp is enabled
+- 125/250/500kHz LoRa demodulator
+- (G)FSK demodulator
+- Direct interface to Semtech transceivers
+- SX1255, SX1257 and SX1250
+- Single 32MHz clock
+
+Read more about the SX1303 on Semtech's SX1303 [product page](https://www.semtech.com/products/wireless-rf/lora-core/sx1303).
+
+## SX1308
+
+{{< figure src="SX1308.jpg" class="float plain" >}}
+
+The SX1308 is the lower powered concentrator compared to the SX1301/2/3. It is designed for indoor gateways.
```

Comment:
...is a lower power concentrator than the SX1301...

Is this true, or is this what Semtech says? As far as I know the only difference between the SX1301 and the SX1308 is the noise figure.